crane-cloud/frontend
# Dashboard user following fixes

# Description

This PR aims to fix user following on the dashboard by stopping section loading everytime a user is followed or unfollowed

## Type of change


## Click up Ticket ID

https://app.clickup.com/t/86974uyqy
## How Can This Been Tested?

Navigate to the suggested users section on the dashboard. Follow/unfollow users and observe.


## Checklist:
